src/examples_localhost/c/rasta.c

```diff
@@ -164,6 +164,33 @@ void onReceive(struct rasta_notification_result *result) {
     }
 }
 
+struct connect_event_data {
+    struct rasta_handle * h;
+    struct RastaIPData * ip_data_arr;
+    fd_event * connect_event;
+    fd_event * schwarzenegger;
```

Comment:
What's the semantics of this? 😄 
Edit: ah, `terminator`, I see -- maybe choose something more obvious?